packages/react/src/components/Tile/next/Tile.js

```diff
@@ -0,0 +1,692 @@
+
+  /**
+   * Specify whether the SelectableTile should be disabled
+   */
+  disabled: PropTypes.bool,
+
+  /**
+   * Specify the function to run when the SelectableTile is clicked
+   */
+  handleClick: deprecate(
+    PropTypes.func,
+    'The `handleClick` prop for `SelectableTile` has been deprecated in favor of `onClick`. It will be removed in the next major release.'
+  ),
+
+  /**
+   * Specify the function to run when the SelectableTile is interacted with via a keyboard
+   */
+  handleKeyDown: deprecate(
+    PropTypes.func,
+    'The `handleKeyDown` prop for `SelectableTile` has been deprecated in favor of `onKeyDown`. It will be removed in the next major release.'
+  ),
+
+  /**
+   * The description of the checkmark icon.
+   */
+  iconDescription: deprecate(
+    PropTypes.string,
+    'The `iconDescription` prop for `SelectableTile` is no longer needed and has ' +
+      'been deprecated. It will be removed in the next major release.'
+  ),
+
+  /**
+   * The ID of the `<input>`.
+   */
+  id: PropTypes.string,
+
+  /**
+   * `true` to use the light version. For use on $ui-01 backgrounds only.
+   * Don't use this to make tile background color same as container background color.
+   */
+  light: PropTypes.bool,
+
+  /**
+   * The `name` of the `<input>`.
+   */
+  name: PropTypes.string,
+
+  /**
+   * The empty handler of the `<input>`.
+   */
+  onChange: PropTypes.func,
+
+  /**
+   * Specify the function to run when the SelectableTile is clicked
+   */
+  onClick: PropTypes.func,
+
+  /**
+   * Specify the function to run when the SelectableTile is interacted with via a keyboard
+   */
+  onKeyDown: PropTypes.func,
+
+  /**
+   * `true` to select this tile.
+   */
+  selected: PropTypes.bool,
+
+  /**
+   * Specify the tab index of the wrapper element
+   */
+  tabIndex: PropTypes.number,
+
+  /**
+   * The `title` of the `<input>`.
+   */
+  title: PropTypes.string,
+
+  /**
+   * The value of the `<input>`.
+   */
+  value: PropTypes.oneOfType([PropTypes.string, PropTypes.number]).isRequired,
+};
+
+export class ExpandableTile extends Component {
+  state = {};
+
+  static propTypes = {
+    /**
+     * The child nodes.
+     */
+    children: PropTypes.node,
+
+    /**
+     * The CSS class names.
+     */
+    className: PropTypes.string,
+
+    /**
+     * `true` if the tile is expanded.
+     */
+    expanded: PropTypes.bool,
+
+    /**
+     * Deprecated in v11. Use 'onClick' instead.
+     */
+    handleClick: deprecate(
+      PropTypes.func,
+      'The handleClick prop for ClickableTile has been deprecated in favor of onClick. It will be removed in the next major release.'
+    ),
+
+    /**
+     * An ID that can be provided to aria-labelledby
+     */
+    id: PropTypes.string,
+
+    /**
+     * `true` to use the light version. For use on $ui-01 backgrounds only.
+     * Don't use this to make tile background color same as container background color.
+     */
+    light: PropTypes.bool,
```

Comment:
I think we wanna deprecate this, since we're not supporting `light` anymore in v11? in favor of the `Layer` component